orttraining/orttraining/training_ops/cpu/loss/softmax_cross_entropy_loss.cc

```diff
@@ -0,0 +1,369 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I took a closer look at the CUDA kernel, so I am skipping this CPU impl, as we are not using this kernel in the production. 
I hope the UT is guard this.

Reply:
Thanks, yes, we have 40 some UTs that verify CPU output matches GPU, so function correctness is there. 

---
In reply to: [408668665](https://github.com/microsoft/onnxruntime/pull/3465#discussion_r408668665) [](ancestors = 408668665)